.gitignore

```
@@ -1,3 +1,4 @@
 env/
 *.log
 *.pyc
+*.DS_Store
```

Still showing up in subdirectories of `sql`
```suggestion
**/.DS_Store
```

https://stackoverflow.com/questions/18393498/gitignore-all-the-ds-store-files-in-every-folder-and-subfolder